prep-javascript-variables/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = 'Avallon W. Graham - Smith';
+    console.log('fullName:', fullName);
+
+    var numberOfPets = '10';
+    console.log('numberOfPets:', numberOfPets);
+
+    const likesVegetables = 'True';
+    console.log('likesVegtables:', likesVegetables);
+
+    var nothing = 'null';
```

Comment:
For numbers, booleans, and `null`, you'll want to leave out the quotes so that the JavaScript engine doesn't interpret them as string values.

Reply:
![image](https://user-images.githubusercontent.com/96964291/150027477-e279ee74-00ae-4a48-a08a-db193b592d86.png)


Quotes have been removed from the numbers and the null. (A silly mistake, pardon my ignorance haha)
When I removed the quotations from the boolean "True", it didn't work in the web browser. Please let me know if this is correct, thank you!